tpu_submit.sh

```diff
@@ -0,0 +1,62 @@
+export OPENBLAS_NUM_THREADS="1"
+export MKL_NUM_THREADS="1"
+
+#$1 = tpu/gpu (def tpu)
+#$2 = train + eval_dev or train only (def train)
+#$3 = class_name (def Librispeech960Base)
+
+HOME_BASE="/home/anjali/data/mlcommons/librispeech/models/wer"
+# GS_BASE="gs://the-peoples-speech-west-europe/ag/ctc_librispeech/training_logs"
+GS_BASE="gs://the-peoples-speech-west-europe/"
+
+DATE=$(date '+log_%Y_%m_%d_%H')
+# FLDRDATE=$(date '+%m%d/%H%M')
+# FLDRDATE=$(date '+%m%d')
+FLDRDATE=$4
+CLS=$3
+
+# if tpu- in $1
+if [[ "$1" == *"-tpu"* ]]; then
+    LOGDIR="${GS_BASE}/${FLDRDATE}/${CLS}"
+
+    name=$1
+    ip_addr=$(../ctpu st -name ${name} --details | grep "TPU IP" | grep -oP "10.*")
+
+    if [ -z "$ip_addr" ]; then
+        echo "Couldnt find TPU, creating a new one"
+        ../ctpu up -name ${name} -tpu-only -tpu-size v3-8 -tf-version 2.2
```

Comment:
You don't need to change this, but just a note, this assumings the path of ctpu.